File: Models/Staff.cs
using System.Drawing.Printing;

namespace fullstackCsharp.Models
{
   
    public class Staff
    {
        private string id_nv;
        private string namenv;
        private string gender;
        private string phone;
        private string address;
        private string user;
        private string password;
        private int rank;
        private string status;
        private string id_pb;
        private string id_rank;



        public Staff()
        {
        }

        public Staff(string id_nv, string namenv, string gender, string phone, string address, string user, string password, int rank, string status, string id_pb, string id_rank)
        {
            this.id_nv = id_nv;
            this.namenv = namenv;
            this.gender = gender;
            this.phone = phone; 
            this.address = address;
            this.user = user;
            this.password = password;
            this.rank = rank;
            this.status = status;
            this.id_pb = id_pb;
            this.Id_rank = id_rank;


        }

        public string Id_nv { get => id_nv; set => id_nv = value; }
        public string Namenv { get => namenv; set => namenv = value; }
        public string Gender { get => gender; set => gender = value; }
        public string Phone { get => phone; set => phone = value; }
        public string Address { get => address; set => address = value; }
        public string User { get => user; set => user = value; }
        public string Password { get => password; set => password = value; }
        public int Rank { get => rank; set => rank = value; }
        public string Status { get => status; set => status = value; }
        public string Id_pb { get => id_pb; set => id_pb = value; }
        public string Id_rank { get => id_rank; set => id_rank = value; }








        //public Staff(int)
    }

}

File: wwwroot/js/Staff.js

//====================================================================== Edit ======================================
let IdNV;
let NameNV;
let gender;
let Address;
let User;
let Password;
let objDS = {
    IdNV: IdNV,
    NameNV: NameNV,
    gender: gender,
    Address: Address,
    User: User,
    Password: Password

}

function conf1(id) {

    console.log(event.target);
    console.log(id);
    document.getElementById("edit1").value = id;
    let willEdit = confirm("Bạn có chắc chắn muốn xóa nhân viên" + results + "không?");
    if (willEdit) {
        let url = '@Html.Raw(Url.Action("Edit", new { Id = "_Id" }))';
        url = url.replace("_Id", results)
        window.location.href = url;
    }
}

//================================================ Delete ================================================

function confDeleteModal() {

    $("#Deleted").submit();

}
function confDelete() {

    let galleryModal = new bootstrap.Modal(document.getElementById('exampleModal'), {
        keyboard: false
    });

    let checkbox = document.getElementsByName('deleted');
    let result = [];
    for (var i = 0; i < checkbox.length; i++) {
        if (checkbox[i].checked === true) {
            result.push(checkbox[i].value);
        }
    }
    document.getElementById('deleteList').value = result.join();
    let t = result.join();
    let deleteModalBody = document.getElementById('deleteModalBody');
    deleteModalBody.innerHTML = t;
    galleryModal.show();
}
//=========================== button setting================================
function confSetting() {


    //let th = document.createElement("th");
    //let td = $("<td>");
    //let input = $("<input>");
    ////=================== thẻ input =========================================
    //$(input).attr('name', 'deleted');
    //$(input).attr('id', 'delete');
    //$(input).attr('type', 'checkbox');

    //$(input).attr('value', '@staff.Id_nv');
    //$(td).append(input);

    ////=================================================================================
    //td.attr('id', 'checkBoxDelete');

    //$('#popertyTableTh').append(th);
    //$('.popertyTableTd').append(td);


    //$(th).attr('id', 'propertyTh');

    //th.innerHTML = "delete";
    $("#delete").show();
    $("#btnCreateDelete").show();
    
    
    //document.getElementById("btnCreateDelete").style.display = "block";
    //document.getElementByID("btnSetting").style.display = "none" ;
   // $('#btnSetting').attr('class', 'd-none');


}

